martin_callejas/task1_tic-tac-toe/src/jalasoft/training/Game.java

```diff
@@ -155,4 +159,13 @@ private boolean allItemsAreEqual(ArrayList<Spot> line){
         }
         return true;
     }
+
+    public int tryParse(String value){
+        int defaultValue = - 1;
+        try {
+            return Integer.parseInt(value);
+        }catch (NumberFormatException e) {
+            return defaultValue;
```

Comment:
In addition... can be good to have some message? saying that something went wrong because it keeps asking again and again for another input but user will not guess his error.

Reply:
Fair enough/ I've added a message when position is requested. IMO is very intuitive for a user to figure out that a number is needed here.